### PR TITLE
[Pages] refactored direct uploads docs :)

### PR DESCRIPTION
**Pull request by nevikashah**

### CODE REVIEW

File: content/pages/platform/direct-upload.md [hunk 1]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 Once you have your prebuilt assets ready, there are two ways to begin uploading: 
 
 * [Wrangler](/pages/platform/direct-upload/#wrangler-cli).
-* [Drag and drop](/pages/platform/direct-upload/#drag-and-drop).
+* [Drag and Drop](/pages/platform/direct-upload/#drag-and-drop).
 
 {{<Aside type= "note">}}
   
-Within a project, you can switch between creating deployments with either method. However, you cannot push additional changes through your standard Git integration within the same project if that repository is already using Direct Uploads.
+Within a direct uploads project, you can switch between creating deployments with either Wrangler or Drag and Drop. However, you cannot create deployments with these methods on a standard Git integrated project.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Within a direct uploads project, you can switch between creating deployments with either Wrangler or Drag and Drop. However, you cannot create deployments with these methods on a standard Git integrated project.
```

File: content/pages/platform/direct-upload.md [hunk 2]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 
 {{</Aside>}}
 
 ## Supported file types
 
 Below is the supported file types for each Direct Upload options:
-* Wrangler: A single folder of assets(for example, HTML, CSS, JS, PNG, SVG).
-* Drag and drop: Zip or folder of assets(for example, HTML, CSS, JS, PNG, SVG).
+* Wrangler: A single folder of assets
+* Drag and drop: A zip file or single folder of assets
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
* Drag and Drop: A zip file or single folder of assets.
```
Capitalization consistency.

File: content/pages/platform/direct-upload.md [hunk 3]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
+Within a direct uploads project, you can switch between creating deployments with either Wrangler or Drag and Drop. However, you cannot create deployments with these methods on a standard Git integrated project. 
 
 {{</Aside>}}
 
 ## Supported file types
 
 Below is the supported file types for each Direct Upload options:
-* Wrangler: A single folder of assets(for example, HTML, CSS, JS, PNG, SVG).
-* Drag and drop: Zip or folder of assets(for example, HTML, CSS, JS, PNG, SVG).
+* Wrangler: A single folder of assets
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
* Wrangler: A single folder of assets. (Zip files are not supported.)
```

File: content/pages/platform/direct-upload.md [hunk 4]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 ### Set up Wrangler
 
-To begin, [install and set up the latest version of Wrangler](/workers/wrangler/get-started/). Note that Pages integration with Wrangler relies on Wrangler 2.
+To begin, [install and set up the latest version of Wrangler](/workers/wrangler/get-started/). Note that Pages relies on Wrangler 2.
 
-#### Deploy your project with Wrangler
+#### Create your project
 
-Run the following Wrangler command to create a project: 
+Once you have logged into Wrangler, it’s time to create your first project. Execute the following command:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Log in to Wrangler with the `wrangler login` command then run the following command:
```

File: content/pages/platform/direct-upload.md [hunk 5]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 
 ```sh
-wrangler pages publish <project directory>
+$ wrangler pages project create
 ```
 
-After running `wrangler pages publish`, you will be prompted to choose whether you would like to publish assets for an existing project or if you would like to create a new one. To begin a new project, select create a new project, continue to name your project, and deploy. Subsequent deployments will reuse these values (saved in your `node_modules/.cache/wrangler` folder).
+You will then be prompted to specify the project name. Your project will be served at `<project-name>.pages.dev`. You will also be prompted to specify your production branch.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
You will then be prompted to specify the project name. Your project will be served at `<PROJECT_NAME>.pages.dev`. You will also be prompted to specify your production branch. 
```

File: content/pages/platform/direct-upload.md [hunk 6]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
-After running `wrangler pages publish`, you will be prompted to choose whether you would like to publish assets for an existing project or if you would like to create a new one. To begin a new project, select create a new project, continue to name your project, and deploy. Subsequent deployments will reuse these values (saved in your `node_modules/.cache/wrangler` folder).
+You will then be prompted to specify the project name. Your project will be served at `<project-name>.pages.dev`. You will also be prompted to specify your production branch. 
 
-After you deploy your Project, go to your newly created Pages project in the Cloudflare dashboard to access deployment details, including its shareable and unique preview URL. 
+Subsequent deployments will reuse both of these values (saved in your `node_modules/.cache/wrangler` folder).
 
-#### Create a new deployment
 
-After you have deployed your project, you can continue to add new deployments to that project. Deployments will be available at the following convention: `<DEPLOYMENT>.<PROJECT_NAME>.pages.dev`. 
+#### Publish your assets
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
#### Publish your assets

```

File: content/pages/platform/direct-upload.md [hunk 7]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 
-#### Create a new deployment
 
-After you have deployed your project, you can continue to add new deployments to that project. Deployments will be available at the following convention: `<DEPLOYMENT>.<PROJECT_NAME>.pages.dev`. 
+#### Publish your assets 
+From here, you have created an empty project and can now publish your assets for your first deployment and for all subsequent deployments in your production environment. To do this, execute the following command:
 
 ```sh
-wrangler pages publish <DIRECTORY> --branch=[BRANCH]
+$ wrangler pages publish <output-directory>
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
$ wrangler pages publish <OUTPUT_DIRECTORY>
```

File: content/pages/platform/direct-upload.md [hunk 8]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 ```
 
+Your production deployment will be available at `<project-name>.pages.dev`.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Your production deployment will be available at `<PROJECT_NAME>.pages.dev`.
```

File: content/pages/platform/direct-upload.md [hunk 9]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
+  
 {{<Aside type= "note">}}
 
-If you are in a Git workspace, Wrangler will automatically pull the branch information for you. Otherwise, you will be prompted to choose your branch (which will then determine if the deployment is production or preview).
+Before using the publish command, you will need to make sure you are inside the project. If not, you can also pass in the project path.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Before using the `wrangler publish` command, you will need to make sure you are inside the project. If not, you can also pass in the project path. 
```

File: content/pages/platform/direct-upload.md [hunk 10]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 
+Your production deployment will be available at `<project-name>.pages.dev`.
+  
 {{<Aside type= "note">}}
 
-If you are in a Git workspace, Wrangler will automatically pull the branch information for you. Otherwise, you will be prompted to choose your branch (which will then determine if the deployment is production or preview).
+Before using the publish command, you will need to make sure you are inside the project. If not, you can also pass in the project path. 
 
 {{</Aside>}}
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md:63]:
```suggestion
```

File: content/pages/platform/direct-upload.md [hunk 11]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
+Your production deployment will be available at `<project-name>.pages.dev`.
+  
 {{<Aside type= "note">}}
 
-If you are in a Git workspace, Wrangler will automatically pull the branch information for you. Otherwise, you will be prompted to choose your branch (which will then determine if the deployment is production or preview).
+Before using the publish command, you will need to make sure you are inside the project. If not, you can also pass in the project path. 
 
 {{</Aside>}}
+ 
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
 
```

File: content/pages/platform/direct-upload.md [hunk 12]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
+#### Publish your assets 
+From here, you have created an empty project and can now publish your assets for your first deployment and for all subsequent deployments in your production environment. To do this, execute the following command:
 
 ```sh
-wrangler pages publish <DIRECTORY> --branch=[BRANCH]
+$ wrangler pages publish <output-directory>
 ```
 
+Your production deployment will be available at `<project-name>.pages.dev`.
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
 
```

File: content/pages/platform/direct-upload.md [hunk 13]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
-If you are in a Git workspace, Wrangler will automatically pull the branch information for you. Otherwise, you will be prompted to choose your branch (which will then determine if the deployment is production or preview).
+Before using the publish command, you will need to make sure you are inside the project. If not, you can also pass in the project path. 
 
 {{</Aside>}}
+ 
+  
+However, to publish assets to a preview environment, you can execute: 
+
+```sh
+$ wrangler pages publish <output-directory> --branch=<branch-name>
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
$ wrangler pages publish <OUTPUT_DIRECTORY> --branch=<BRANCH_NAME>
```

File: content/pages/platform/direct-upload.md [hunk 14]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
+  
 {{<Aside type= "note">}}
 
-If you are in a Git workspace, Wrangler will automatically pull the branch information for you. Otherwise, you will be prompted to choose your branch (which will then determine if the deployment is production or preview).
+Before using the publish command, you will need to make sure you are inside the project. If not, you can also pass in the project path. 
 
 {{</Aside>}}
+ 
+  
+However, to publish assets to a preview environment, you can execute:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
However, to publish assets to a preview environment, run: 
```

File: content/pages/platform/direct-upload.md [hunk 15]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
+ 
+  
+However, to publish assets to a preview environment, you can execute: 
+
+```sh
+$ wrangler pages publish <output-directory> --branch=<branch-name>
+```
+
+For every branch you create, a branch alias will be available to you at `<branch-name>.<project-name>.pages.dev`. 
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion

For every branch you create, a branch alias will be available to you at `<BRANCH_NAME>.<PROJECT_NAME>.pages.dev`. 

```

File: content/pages/platform/direct-upload.md [hunk 16]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
+{{<Aside type= "note">}}
+
+If you are in a Git workspace, Wrangler will automatically pull the branch information for you. Otherwise, you will need to specify your branch in this command.
+
+{{</Aside>}}
+
+If you would like to streamline the project creation and asset publishing steps, you can also use the publish command to both create and publish assets at the same time. If you execute this command first, you will still be prompted to specify your project name and production branch. These values will still be cached for subsequent deployments as stated above. If the cache already exists and you would like to create a new project, you will need to run the [create command](#create-your-project).
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
If you would like to streamline the project creation and asset publishing steps, you can also use the publish command to both create and publish assets at the same time. If you execute this command first, you will still be prompted to specify your project name and production branch. These values will still be cached for subsequent deployments as stated above. If the cache already exists and you would like to create a new project, you will need to run the [`create` command](#create-your-project). 
```

File: content/pages/platform/direct-upload.md [hunk 17]
```diff
@@ -84,17 +103,20 @@ To deploy with drag and drop:
 2. In **Account Home**, select your account > **Pages**.
 3. Select **Create a project** > **Direct Upload** > enter your project name in the provided field > **Create project**. 
 
-Your project will be served from `<PROJECT_NAME>.pages.dev`. Next drag and drop your build output directory into the uploading frame. Once your files have been successfully uploaded, select **Save and Deploy** and continue to your newly deployed project. 
+Your project will be served from `<project-name>.pages.dev`. Next drag and drop your build output directory into the uploading frame. Once your files have been successfully uploaded, select **Save and Deploy** and continue to your newly deployed project.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Your project will be served from `<PROJECT_NAME>.pages.dev`. Next drag and drop your build output directory into the uploading frame. Once your files have been successfully uploaded, select **Save and Deploy** and continue to your newly deployed project. 
```
all caps for input values is a CF style guide convention

File: content/pages/platform/direct-upload.md [hunk 18]
```diff
@@ -84,17 +103,20 @@ To deploy with drag and drop:
...
 
-After you have your project created, select **Create a new deployment** to begin a new version of your site. Next, choose whether your new deployment will be made to your production or preview environment. If choosing preview, you can create a new deployment target or enter an existing one. Deployment targets allow you to access all changes at one preview subdomain with the following convention: `<DEPLOYMENT_TARGET>.<PROJECT_NAME>.pages.dev`. 
+After you have your project created, select **Create a new deployment** to begin a new version of your site. Next, choose whether your new deployment will be made to your production or preview environment. If choosing preview, you can create a new deployment branch or enter an existing one. 
 
 ## Troubleshoot
 
 ### Limits
+In regards to file count: 
+* For Wrangler uploads: there is a 20,000 file limit
+* For Drag and Drop uploads: there is a 1,000 file limit
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion

Regarding file count:

* For Wrangler uploads, there is a 20,000 file limit.
* For Drag and Drop uploads, there is a 1,000 file limit.
```

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]

File: content/pages/platform/direct-upload.md (hunk 5 above)
Comment (isaac-mcfadyen) [reply content/pages/platform/direct-upload.md]: Is this accurate? If the name is already taken Pages appends a random 3 letter/number string to the end, should that be mentioned?

**Review by deadlypants1973 — APPROVED** [review]

File: content/pages/platform/direct-upload.md [hunk 19]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
 
 # Direct Uploads
 
-With Direct Uploads, you can bring your prebuilt assets right to Pages. By using your own continuous integration (CI) tooling to handle the build, deploy your website's assets right to the Cloudflare edge network.
+With Direct Uploads, you can bring your prebuilt assets right to Pages. Bring your own Continuous Integration (CI) tooling to handle the build and deploy your website's assets straight to the Cloudflare edge network.
 
 ## Methods to uploading
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
## Upload methods
```

File: content/pages/platform/direct-upload.md [hunk 20]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
 
 # Direct Uploads
 
-With Direct Uploads, you can bring your prebuilt assets right to Pages. By using your own continuous integration (CI) tooling to handle the build, deploy your website's assets right to the Cloudflare edge network.
+With Direct Uploads, you can bring your prebuilt assets right to Pages. Bring your own Continuous Integration (CI) tooling to handle the build and deploy your website's assets straight to the Cloudflare edge network.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
With Direct Uploads, you can bring your prebuilt assets directly to Pages. Bring your own Continuous Integration (CI) tooling to handle the build and deploy your website's assets straight to the Cloudflare edge network.
```

File: content/pages/platform/direct-upload.md [hunk 21]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
 
 # Direct Uploads
 
-With Direct Uploads, you can bring your prebuilt assets right to Pages. By using your own continuous integration (CI) tooling to handle the build, deploy your website's assets right to the Cloudflare edge network.
+With Direct Uploads, you can bring your prebuilt assets directly to Pages. Bring your own Continuous Integration (CI) tooling to handle the build and deploy your website's assets straight to the Cloudflare edge network.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]: This opener needs to be rewritten to state:
1. what is direct uploads
2. how it works
3. what this guide will instruct developer to do

File: content/pages/platform/direct-upload.md [hunk 22]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 
-## Methods to uploading
+## Upload methods
 
 Once you have your prebuilt assets ready, there are two ways to begin uploading:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
After you have your prebuilt assets ready, there are two ways to begin uploading: 
```

File: content/pages/platform/direct-upload.md [hunk 23]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 Once you have your prebuilt assets ready, there are two ways to begin uploading: 
 
 * [Wrangler](/pages/platform/direct-upload/#wrangler-cli).
-* [Drag and drop](/pages/platform/direct-upload/#drag-and-drop).
+* [Drag and Drop](/pages/platform/direct-upload/#drag-and-drop).
 
 {{<Aside type= "note">}}
   
-Within a project, you can switch between creating deployments with either method. However, you cannot push additional changes through your standard Git integration within the same project if that repository is already using Direct Uploads.
+Within a direct uploads project, you can switch between creating deployments with either Wrangler or Drag and Drop. However, you cannot create deployments with these methods on a standard Git integrated project.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Within a Direct Uploads project, you can switch between creating deployments with either Wrangler or Drag and Drop. However, you cannot create deployments with Direct Uploads on a project that you created through Git integration on the dashboard. Only projects created with Direct Uploads can be updated with Direct Uploads.
```

Comment (deadlypants1973) [reply content/pages/platform/direct-upload.md]: What is the second sentence saying?

File: content/pages/platform/direct-upload.md [hunk 24]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 
-With Direct Uploads, you can bring your prebuilt assets right to Pages. By using your own continuous integration (CI) tooling to handle the build, deploy your website's assets right to the Cloudflare edge network.
+With Direct Uploads, you can bring your prebuilt assets directly to Pages. Bring your own Continuous Integration (CI) tooling to handle the build and deploy your website's assets straight to the Cloudflare edge network.
 
-## Methods to uploading
+## Upload methods
 
 Once you have your prebuilt assets ready, there are two ways to begin uploading: 
 
 * [Wrangler](/pages/platform/direct-upload/#wrangler-cli).
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md:14]:
```suggestion
* [Wrangler](/pages/platform/direct-upload/#wrangler-cli): the Workers CLI.
```

File: content/pages/platform/direct-upload.md [hunk 25]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
 
-#### Deploy your project with Wrangler
+#### Create your project
 
-Run the following Wrangler command to create a project: 
+Log in to Wrangler with the `wrangler login` command then run the following command:
 
 ```sh
-wrangler pages publish <project directory>
+$ wrangler pages project create
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md:40]: There is no need to specify a directory right? specifically what is the developer not able to do?

File: content/pages/platform/direct-upload.md [hunk 26]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
+With Direct Uploads, you can bring your prebuilt assets directly to Pages. Bring your own Continuous Integration (CI) tooling to handle the build and deploy your website's assets straight to the Cloudflare edge network.
 
-## Methods to uploading
+## Upload methods
 
 Once you have your prebuilt assets ready, there are two ways to begin uploading: 
 
 * [Wrangler](/pages/platform/direct-upload/#wrangler-cli).
-* [Drag and drop](/pages/platform/direct-upload/#drag-and-drop).
+* [Drag and Drop](/pages/platform/direct-upload/#drag-and-drop).
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md:15]:
```suggestion
* [Drag and Drop](/pages/platform/direct-upload/#drag-and-drop): 
```

Comment (deadlypants1973) [reply content/pages/platform/direct-upload.md:15]: This needs a short description. Also is drag and drop even included in this guide?

File: content/pages/platform/direct-upload.md [hunk 27]
```diff
@@ -88,13 +107,18 @@ Your project will be served from `<PROJECT_NAME>.pages.dev`. Next drag and drop
...
 
-There is a 1,000 file [limits](/pages/platform/limits/#files) in place for both drag and drop and Wrangler upload in regards to file count and a 25 MiB limit in place for individual file size. 
+Regarding file count:
+
+* For Wrangler uploads, there is a 20,000 file limit.
+* For Drag and Drop uploads, there is a 1,000 file limit.
+
+On both upload methods, there is a 25 MiB limit in place for individual file size. 
 
 If using the drag and drop method, a red warning symbol will appear next to an asset if too large and thus unsuccessfully uploaded. In this case, you may choose to delete that asset but you cannot replace it. In order to do so, you must reupload the entire project.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
If using the Drag and Drop method, a red warning symbol will appear next to an asset if too large and thus unsuccessfully uploaded. In this case, you may choose to delete that asset but you cannot replace it. In order to do so, you must reupload the entire project.
```

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]

File: content/pages/platform/direct-upload.md [hunk 28]
```diff
@@ -5,71 +5,90 @@ title: Direct Uploads
...
-Run the following Wrangler command to create a project: 
+Log in to Wrangler with the `wrangler login` command then run the following command:
 
 ```sh
-wrangler pages publish <project directory>
+$ wrangler pages project create
 ```
 
-After running `wrangler pages publish`, you will be prompted to choose whether you would like to publish assets for an existing project or if you would like to create a new one. To begin a new project, select create a new project, continue to name your project, and deploy. Subsequent deployments will reuse these values (saved in your `node_modules/.cache/wrangler` folder).
+You will then be prompted to specify the project name. Your project will be served at `<PROJECT_NAME>.pages.dev`. You will also be prompted to specify your production branch.
```
Comment (isaac-mcfadyen) [inline content/pages/platform/direct-upload.md]:
```suggestion
You will then be prompted to specify the project name. Your project will be served at `<PROJECT_NAME>.pages.dev` (or your project name plus a few random characters if your project name is already taken). You will also be prompted to specify your production branch. 
```

File: content/pages/platform/direct-upload.md (hunk 21 above)
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Direct Uploads enable you to upload your prebuilt assets to the Pages platform and deploy them to the Cloudflare edge network. This guide will instruct you how to upload your assets using Wrangler or the Drag and Drop method.
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**